rero/rero-ils-ui
# acquisitions: implement views

* Adds brief view
* Adds detail view
* Adds entry on routing

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

## Why are you opening this PR?

- Implements brief and detail view for Acquisition account

## How to test?

- Checkout PR acquistion: create acq_account resource: https://github.com/rero/rero-ils/pull/669
- `bootstrap`and `setup`
- Log with `system_librarian`
- Check new entry acquisition > account (add, update, delete)
- Check filter list with `librarian` role

## Code review check list
